tests/debug-github-links.test.ts

```
@@ -0,0 +1,80 @@
+import { UiComponent } from '../src/ui/uiComponent';
+import { createBasicFixture } from '../src/utils/test/fixtures';
+import { loadFromEnv } from '../src/utils/util';
+
+const test = createBasicFixture();
+const test = createBasicFixture();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'test'. Consider to delete it or rename it.

'test' is defined here:


(lint/suspicious/noRedeclare)